PNChart/PNBarChartGrouped.h

```diff
@@ -10,11 +10,6 @@
 #import "PNChartDelegate.h"
 #import "PNBar.h"
 
-#define xLabelMargin 15
-#define yLabelMargin 15
```

Comment:
@mariabernis where did the margins go? Can't find them. Is that an error or they weren't needed?